terraform-aws/variables.tf

```diff
@@ -57,6 +81,10 @@ variable "worker_instance_type" {
   default = "c5.4xlarge"
 }
 
+variable "client_instance_type" {
+  default = "c5.4xlarge"
```

Comment:
That's way too large for VMs running web apps. Can safely be a 2 or 4 core machine with 8GB memory